glibc.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: glibc
-  version: 2.40
-  epoch: 8
+  version: "2.41"
+  epoch: 0
```

Comment:
@sil2100 check if 0 was already used/burned/withdrawn, try not to use it again as it will fail to publish again.